src/_guides/language/concurrency/index.md

```diff
@@ -312,107 +311,144 @@ to implement isolates.
 
  [Flutter `compute()` function]: {{site.flutter-docs}}/cookbook/networking/background-parsing#4-move-this-work-to-a-separate-isolate
 
-
 ### Implementing a simple worker isolate
 
-This section shows the implementation for a
-main isolate and the simple worker isolate that it spawns.
-The worker isolate executes a function and then exits,
-sending the main isolate a single message as it exits.
-(The [Flutter `compute()` function][] works in a similar way.)
+These examples show the implementation of a
+main isolate that spawns a simple worker isolate.
+The examples use [`Isolate.run()`][], 
+which efficiently abstracts the complexity
+of setting up and managing worker isolates manually:
 
-This example uses the following isolate-related API:
+* Spawns (starts and creates) an isolate
+* Takes a function argument and executes
+* Captures the result
+* Returns the result to the main isolate
+* Terminates the isolate once work is complete
+* Checks, captures, and throws exceptions and errors back to the main isolate
 
-* [`Isolate.spawn()`][] and [`Isolate.exit()`][]
-* [`ReceivePort`][] and [`SendPort`][]
+[`Isolate.run()`]: {{site.dart-api}}/dev/dart-isolate/Isolate/run.html
 
-[`Isolate.exit()`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/Isolate/exit.html
-[`Isolate.spawn()`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/Isolate/spawn.html
-[`ReceivePort`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/ReceivePort-class.html
-[`SendPort`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/SendPort-class.html
+#### Running an existing method in a new isolate
 
 Here’s the code for the main isolate:
```

Comment:
**suggestion:** Don't use "here" to indicate location. "Here" would be that exact line. It's not "here", the code _follows_ this block. I know that it's good to have copy before an example and after a heading, but make sure it adds value.
```suggestion
You can write a main isolate that resembles the following code:
```

**question:** What makes this the "main" isolate? Is it because it's in the `main` class? Does `main` = "first"? I would recommend clarifying this here and under **The main isolate**.

Reply:
The main isolate is the isolate that every Dart program run in by default, so yes, the "first" or "starting" (defined higher up [on the same page](https://dart.dev/guides/language/concurrency#the-main-isolate)). I'll go through the page and make sure the terminology is consistent, thanks for putting it on my radar!